### DISCUSSION TITLE
Set different colors in panels

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#584] [Q&A] [answered]
For better visibility while driving, it would be very helpful if the panels for speed, arrival time, altitude etc.
would have different colors.
You wouldn't have to look and search so long for the right panel.
Would be a security aspect.

**Comment by WolfgangZ42** [comment]
Please, please only for the people who like it particularly colorful - so adjustable ... 
I know where my panels are and can easily assign everything using the symbols.

**Reply by Leeb64** [reply]
Up to a certain number of panels this works more or less well.
But above a certain number you start searching..
Colors would be very helpful!
Selectable colors would of course be the optimal.

**Comment by devemux86** [comment]
@Leeb64 thanks for the suggestion.

Do you mean the background color of the panels or the color of the icon or text?

**Reply by Leeb64** [reply]
The color of the symbol + the text would be better, I think.

**Comment by devemux86** [comment]
We could offer options to set different colors in panels.
(Settings | Application | Appearance)

- Customize the colors in the app info panels individually.

- Customize the colors in the POI panels individually.
POI colors are also used in map icons (like with favorites).

- The favorite panels already use their own custom colors.

For those who want to use these options,
they can take the customization of Cruiser's appearance to the next level.

<img src="https://github.com/user-attachments/assets/662089da-fbb1-4bb3-b8f2-b216ddb586d3" width="50%"/>

**Reply by Leeb64** [reply]
Das Ablesen eines bestimmten Wertes geht ,farbig gekennzeichnet, wesentlich schneller-und ist somit ist das ein Beitrag zur Sicherheit!

**Reply by FishEy3** [reply]
This option is very much appreciated, I use the colors with the favorites and it is very functional, thanks if you will implement it

> We could offer options to set different colors in panels.
> 
>     * Customize the colors in the app info panels individually.
> 
>     * Customize the colors in the POI panels individually.
>       POI colors are also used in map icons (like with favorites).
> 
>     * The favorite panels already use their own custom colors.

**Comment by devemux86** [comment]
The appearance of the panels will be controlled by switches to simplify the use of the app.

For different options in navigation and follow location, it is better to use different app profiles.

All panels are needed when moving, so they will only be visible when navigation or follow are active.
Except perhaps for GPS info and Compass accuracy?

<img src="https://github.com/user-attachments/assets/1f46fda3-6774-40cf-8625-d211a54f7f42" width="50%"/>

**Reply by kds112** [reply]
When hiking I normaly use map view without follow or navigation mode. Therefore, time, battery, altitude etc. are useful. Can't you use the switches/profiles for all situations (view, follow, navigation)? It would bei easier for all to understand the mechanism...

**Reply by devemux86** [reply]
@kds112 Use a map app while moving but without a follow location mode?

Then user location will be quickly outside the visible screen.

**Reply by kds112** [reply]
While hiking I only switch my smartphone on if I have to decide to go the left or right way. After this quick view I switch it off again to save battery. I don't want to carry a heavy powerbank in addition. Therefore Follow Mode does not make much sense. But I want to see my selected panels in the map view.
Please do not remove them in map view.

**Reply by devemux86** [reply]
The info panels can be displayed always, like altitude, satellites, compass, time, battery.

The other panels already only measure values when navigation or follow are active.

**Comment by devemux86** [comment]
Another idea is to have different colors on the large direction icon (top left)
depending on the distance from next instruction. For example:
- Very far away: gray direction
- Far away: yellow direction
- Near: green direction

**Comment by devemux86** [comment, marked as answer]
- Different colors in panels are available in https://github.com/devemux86/cruiser/discussions/834

(Settings | Application | Appearance)
